packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/SimulationIntermediateResults.java

```diff
@@ -0,0 +1,32 @@
+package software.uncharted.terarium.hmiserver.models;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+import java.io.Serializable;
+
+@Data
+@Accessors(chain = true)
+public class SimulationIntermediateResults implements Serializable {
+	@JsonAlias("job_id")
+	private String jobId;
+	private float progress;
+
+	public void setJobId(String jobId){
+		this.jobId = jobId;
+	}
+
+	public void setProgress(float progress){
+		this.progress = progress;
+	}
+
+	public String getJobId(){
+		return this.jobId;
+	}
```

Comment:
Isn't `@Data` supposed to handle those @dvince2?

Reply:
I thought so too but I couldnt use these functions until I created them myself 
Maybe I have the naming wrong for @data? Im not sure

Reply:
Strange... yeah the `@Data` annotation should take care of that.  You can use specific `@Setter` and `@Getter` on these fields as well too if not?

Reply:
<img width="609" alt="image" src="https://github.com/DARPA-ASKEM/Terarium/assets/30270901/f1fddc9b-e1c4-4917-9339-9ae80541f289">

<img width="430" alt="image" src="https://github.com/DARPA-ASKEM/Terarium/assets/30270901/7b8ccb5d-a317-4757-83c1-ca694ada34b0">


I'm wondering if you're having an issue with Lombok/your IDE not showing the suggestions? When i blew away the setters/getters on your branch things looked ok to me